src/core/src/package_managers/PackageManager.py

```diff
@@ -470,4 +470,8 @@ def set_security_esm_package_status(self, operation, packages):
     @abstractmethod
     def separate_out_esm_packages(self, packages, package_versions):
         pass
-
+    
+    @abstractmethod
+    def get_package_install_expected_avg_time_in_minutes(self):
+        """Retrieves average time to install package in minutes."""
+        pass
```

Comment:
Same comment as the other one about empty lines at the end of each file

Reply:
Addressed it.

Reply:
2 empty lines

Reply:
comment applies everywhere

Reply:
2 empty lines